core/src/lib/CMakeLists.txt

```diff
@@ -151,11 +151,19 @@ target_compile_definitions(
           -DBAREOS_YEAR="${BAREOS_YEAR}"
 )
 
-target_link_libraries(
-  bareos
-  PRIVATE bareosfastlz ${OPENSSL_LIBRARIES} Threads::Threads ${ZLIB_LIBRARIES}
-          ${LZO2_LIBRARIES} ${CAM_LIBRARIES} CLI11::CLI11 xxHash::xxhash
-)
+if(NOT USE_SYSTEM_XXHASH)
+  target_link_libraries(
+    bareos
+    PRIVATE bareosfastlz ${OPENSSL_LIBRARIES} Threads::Threads ${ZLIB_LIBRARIES}
+            ${LZO2_LIBRARIES} ${CAM_LIBRARIES} CLI11::CLI11 xxHash::xxhash
+  )
+else()
+  target_link_libraries(
+    bareos
+    PRIVATE bareosfastlz ${OPENSSL_LIBRARIES} Threads::Threads ${ZLIB_LIBRARIES}
+            ${LZO2_LIBRARIES} ${CAM_LIBRARIES} CLI11::CLI11 PkgConfig:: XXHASH
+  )
+endif()
```

Comment:
`PkgConfig:: XXHASH` does not work.

Also, please just create an alias named `xxHash::xxhash` that points to `PkgConfig::XXHASH` instead of duplicating the whole call to `target_link_libraries()`.